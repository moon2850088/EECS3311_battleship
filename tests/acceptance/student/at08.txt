debug_test(advanced)
fire([A, 5])
bomb([B, 6], [B, 7])