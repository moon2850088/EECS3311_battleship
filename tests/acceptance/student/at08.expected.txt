  state 0 OK -> Start a new game
->debug_test(advanced)
  state 1 OK -> Fire Away!
     1  2  3  4  5  6  7  8  9 10 11 12
  A  _  _  _  _  h  h  h  _  _  _  _  _
  B  _  _  _  _  _  _  _  _  _  _  _  _
  C  _  _  _  _  _  h  h  h  h  h  h  _
  D  _  v  _  _  _  _  _  _  _  _  _  _
  E  _  v  _  _  _  _  _  v  _  _  _  _
  F  _  v  _  v  _  _  _  v  _  _  _  _
  G  _  v  _  v  _  _  _  v  h  _  _  _
  H  _  v  _  v  _  _  _  v  _  _  _  v
  I  _  _  _  v  _  _  _  v  _  _  _  v
  J  _  _  _  _  _  _  _  v  _  _  _  _
  K  _  _  _  _  _  _  _  v  _  _  _  _
  L  _  _  _  _  _  _  _  _  _  _  _  _
  Current Game (debug): 1
  Shots: 0/40
  Bombs: 0/7
  Score: 0/28 (Total: 0/28)
  Ships: 0/7
    7x1: [E, 8]->v;[F, 8]->v;[G, 8]->v;[H, 8]->v;[I, 8]->v;[J, 8]->v;[K, 8]->v
    6x1: [C, 6]->h;[C, 7]->h;[C, 8]->h;[C, 9]->h;[C,10]->h;[C,11]->h
    5x1: [D, 2]->v;[E, 2]->v;[F, 2]->v;[G, 2]->v;[H, 2]->v
    4x1: [F, 4]->v;[G, 4]->v;[H, 4]->v;[I, 4]->v
    3x1: [A, 5]->h;[A, 6]->h;[A, 7]->h
    2x1: [H,12]->v;[I,12]->v
    1x1: [G, 9]->h
->fire([A, 5])
  state 2 OK -> Hit! Keep Firing!
     1  2  3  4  5  6  7  8  9 10 11 12
  A  _  _  _  _  X  h  h  _  _  _  _  _
  B  _  _  _  _  _  _  _  _  _  _  _  _
  C  _  _  _  _  _  h  h  h  h  h  h  _
  D  _  v  _  _  _  _  _  _  _  _  _  _
  E  _  v  _  _  _  _  _  v  _  _  _  _
  F  _  v  _  v  _  _  _  v  _  _  _  _
  G  _  v  _  v  _  _  _  v  h  _  _  _
  H  _  v  _  v  _  _  _  v  _  _  _  v
  I  _  _  _  v  _  _  _  v  _  _  _  v
  J  _  _  _  _  _  _  _  v  _  _  _  _
  K  _  _  _  _  _  _  _  v  _  _  _  _
  L  _  _  _  _  _  _  _  _  _  _  _  _
  Current Game (debug): 1
  Shots: 1/40
  Bombs: 0/7
  Score: 0/28 (Total: 0/28)
  Ships: 0/7
    7x1: [E, 8]->v;[F, 8]->v;[G, 8]->v;[H, 8]->v;[I, 8]->v;[J, 8]->v;[K, 8]->v
    6x1: [C, 6]->h;[C, 7]->h;[C, 8]->h;[C, 9]->h;[C,10]->h;[C,11]->h
    5x1: [D, 2]->v;[E, 2]->v;[F, 2]->v;[G, 2]->v;[H, 2]->v
    4x1: [F, 4]->v;[G, 4]->v;[H, 4]->v;[I, 4]->v
    3x1: [A, 5]->X;[A, 6]->h;[A, 7]->h
    2x1: [H,12]->v;[I,12]->v
    1x1: [G, 9]->h
->bomb([B, 6],[B, 7])
  state 3 OK -> Miss! Keep Firing!
     1  2  3  4  5  6  7  8  9 10 11 12
  A  _  _  _  _  X  h  h  _  _  _  _  _
  B  _  _  _  _  _  O  O  _  _  _  _  _
  C  _  _  _  _  _  h  h  h  h  h  h  _
  D  _  v  _  _  _  _  _  _  _  _  _  _
  E  _  v  _  _  _  _  _  v  _  _  _  _
  F  _  v  _  v  _  _  _  v  _  _  _  _
  G  _  v  _  v  _  _  _  v  h  _  _  _
  H  _  v  _  v  _  _  _  v  _  _  _  v
  I  _  _  _  v  _  _  _  v  _  _  _  v
  J  _  _  _  _  _  _  _  v  _  _  _  _
  K  _  _  _  _  _  _  _  v  _  _  _  _
  L  _  _  _  _  _  _  _  _  _  _  _  _
  Current Game (debug): 1
  Shots: 1/40
  Bombs: 1/7
  Score: 0/28 (Total: 0/28)
  Ships: 0/7
    7x1: [E, 8]->v;[F, 8]->v;[G, 8]->v;[H, 8]->v;[I, 8]->v;[J, 8]->v;[K, 8]->v
    6x1: [C, 6]->h;[C, 7]->h;[C, 8]->h;[C, 9]->h;[C,10]->h;[C,11]->h
    5x1: [D, 2]->v;[E, 2]->v;[F, 2]->v;[G, 2]->v;[H, 2]->v
    4x1: [F, 4]->v;[G, 4]->v;[H, 4]->v;[I, 4]->v
    3x1: [A, 5]->X;[A, 6]->h;[A, 7]->h
    2x1: [H,12]->v;[I,12]->v
    1x1: [G, 9]->h
